src/renderers/shared/hooks/__tests__/ReactComponentTreeHook-test.js

```diff
@@ -185,7 +185,7 @@ describe('ReactComponentTreeHook', () => {
       assertTreeMatches([element, tree]);
     });
 
-    it('uses displayName, name, or Object for factory components', () => {
+    xit('uses displayName, name, or Object for factory components', () => {
```

Comment:
Looks like it's not documented on the jest page for some reason: http://facebook.github.io/jest/docs/api.html#content

cc @cpojer